### DISCUSSION TITLE
Temperature Set Point Resolution change 0.1C to 0.5C

### DISCUSSION THREAD

**Discussion by hnb2907** [jmcollin78/versatile_thermostat#308] [Q&A]
Hi,

I've been using Versatile Thermostat for about a month, and it's working really well for me.  Thanks for your hard work developing it! :)

Maybe I'm missing something in the documentation...
I have created some "thermostat_over_switch" climates.  At the moment, it increments/decrements in 0.1C steps; but I would prefer 0.5C steps.  Is it possible to define the temperature set point resolution?

Reason for request: I have a Sonoff NSPanel running [https://docs.nspanel.pky.eu/stable/card-climate/](url) using its "climate/thermo" card to control the "Versatile Thermostat" climate entities; but it is difficult to operate the UI with 0.1C resolution.

(I notice, that if I create "thermostat_over_climate", and set the underlying climate entity to 0.5C resolution, VT inherits this.)

Many thanks,
Chris.

**Comment by jmcollin78** [comment]
Hello, in over_switch there is no way to control this at now. As you notice in over_climate, the VTherm inherits from the underlying climate.

The only way is to add a new config parameter, in central config and in over_switch parameters.

If someone wants to develop the PR I would be glad to merge it.

The other possibility is to make this configurable into the Versatile Thermostat UI Card without changing anything in the back. I should be very esay to to.

**Comment by jmcollin78** [comment]
See https://github.com/jmcollin78/versatile_thermostat/issues/311#issuecomment-1877033632.
